models/model.js

```diff
@@ -8,8 +8,18 @@ exports.fetchCategories = () =>{
 }
 exports.fetchReviewsByID = (review_id) =>{
     return db.query("SELECT * FROM reviews WHERE review_id = $1;", [review_id]).then(({ rows }) => {
-        console.log(rows)
         if(rows.length) return rows;
         return Promise.reject({status:404, msg: "review_id not found"})
       });
-}
\ No newline at end of file
+}
+
+exports.alterReviewVotes = (review_id, votes_change) =>{
+   return db.query("UPDATE reviews SET votes = votes + $2 WHERE review_id = $1 RETURNING *;", [review_id,votes_change])
+    .then(({ rows }) => {
+        if(rows.length) return rows;
+        if(votes_change === undefined){ 
```

Comment:
It may be better to do this check/reject before we run the query to save ourselves the time and energy it takes to run the query since we have access to this before the query runs